.github/workflows/deploy.yml

```diff
@@ -9,12 +9,12 @@ jobs:
   deploy:
     runs-on: ubuntu-20.04-self-hosted
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
```

Comment:
In fact this change is not related with commit title.
I think one of two would be nice:
- replace the commit title with a more common
- split a commit

Reply:
Changed the commit title and added information in CHANGELOG

Reply:
Let's have this change in a separate commit with a description of why you do it.

Reply:
Done